templates/configmap.yaml

```diff
@@ -19,3 +20,18 @@ data:
   TAIGA_EMAIL_PORT: {{ default 587 .Values.taiga.emailSmtpPort | quote }}
   TAIGA_EMAIL_USER: {{ required "taiga.emailSmtpUser must be set" .Values.taiga.emailSmtpUser | quote }}
 {{- end }}
+{{- if .Values.taiga.ldap.Enabled }}
+  LDAP_AUTH_ENABLE: "True"
+  LDAP_START_TLS: {{ .Values.taiga.ldap.use_tls | default "False" |  quote }}
+  LDAP_SERVER: {{ .Values.taiga.ldap.ldap_server | quote }}
+  LDAP_PORT: {{ .Values.taiga.ldap.port | default 389 | quote }}
+  LDAP_BIND_DN: {{ .Values.taiga.ldap.bind_dn | default "cn=diegoarmandomaradona" |  quote }}
+  LDAP_SEARCH_BASE: {{ .Values.taiga.ldap.base_dn |  quote }}
+  LDAP_BIND_PASSWORD: {{ .Values.taiga.ldap.password |  quote }}
```

Comment:
As you suggested, this should go to secret.yaml

Reply:
Yap. I say: let's merge this and when I have time I will create another pr for this., :)

Reply:
Yep sounds fine